From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Kappler <tkappler@pulumi.com>
Date: Fri, 3 Feb 2023 17:31:18 -0800
Subject: [PATCH] Revert "Update endpointHashIPAddress"

This reverts commit 2197a6c2c7a0ff306cec3432acb9f5680866f034.

This commit causes never-converging diffs due to incorrect use of
a computed property in hashing.

diff --git a/internal/service/route53resolver/endpoint.go b/internal/service/route53resolver/endpoint.go
index 8bbcf1a699..a36639de0a 100644
--- a/internal/service/route53resolver/endpoint.go
+++ b/internal/service/route53resolver/endpoint.go
@@ -433,7 +433,7 @@ func waitEndpointDeleted(ctx context.Context, conn *route53resolver.Client, id s
 func endpointHashIPAddress(v interface{}) int {
 	var buf bytes.Buffer
 	m := v.(map[string]interface{})
-	buf.WriteString(fmt.Sprintf("%s-%s-", m[names.AttrSubnetID].(string), m["ip"].(string)))
+	buf.WriteString(fmt.Sprintf("%s-", m[names.AttrSubnetID].(string)))
 	return create.StringHashcode(buf.String())
 }
 
